src/site/resources/plantuml/architecture.plantuml

```diff
@@ -0,0 +1,21 @@
+@startuml
```

Comment:
@llorllale it's a good idea to use plantuml for diagrams, but keeping `.png` files in the repo is a wrong idea. Find a way to render the file on-fly, they exist.

Reply:
@yegor256 sure